src/Commands/stubs/model.stub

```diff
@@ -3,8 +3,16 @@
 namespace $NAMESPACE$;
 
 use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
 
 class $CLASS$ extends Model
 {
+    use HasFactory;
+
+    protected static function newFactory()
```

Comment:
I think this should probably below the fillable properties, to keep properties on top.